com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -17,11 +17,11 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - `FastBufferWriter` and Reader IsInitialized property. (#1859)
 
 ### Changed
-
+- Changed NetcodeIntegrationTestHelpers to use UnityTransport. (#1870)
 - Updated `UnityTransport` dependency on `com.unity.transport` to 1.0.0 (#1849)
 
 ### Removed
-
+- Removed SIPTransport from integration tests. (#1870)
```

Comment:
we probably do not need to have these changelog entries.

Reply:
NetcodeIntegrationTest is public facing... as well as other integration test related code is public facing too...so it might be just "useful to know"...? (at least that was my logic)

Reply:
was `SIPTransport` ever public? if so, it does make sense to announce its removal.

Reply:
oh wow, apparently it **is** public:
https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/blob/b5f4195c8ce26294ff6f29d9870e3d546628ff92/com.unity.netcode.gameobjects/TestHelpers/Runtime/Transport/SIPTransport.cs#L13